src/hacbs/components/ApplicationDetails/tabs/overview/visualization/nodes/WorkflowNodeTipContent.tsx

```diff
@@ -0,0 +1,103 @@
+
+type WorkflowNodeTipContentProps = {
+  element: Node<NodeModel, WorkflowNodeModelData>;
+};
+
+const WorkflowNodeTipContent: React.FC<WorkflowNodeTipContentProps> = ({ element }) => {
+  const { label, workflowType, children } = element.getData();
+  const { pathname } = useLocation();
+
+  const { elementRef, pipelinesRef } = React.useMemo(() => {
+    const linkData = getLinkForElement(element);
+    const queryParams = `?activeTab=${linkData.tab}${
+      linkData.filter ? `&${linkData.filter.name}=${linkData.filter.value}` : ''
+    }`;
+    return {
+      elementRef: `${pathname}${queryParams}`,
+      pipelinesRef: `${pathname}?activeTab=pipelineruns`,
+    };
+  }, [element, pathname]);
+
+  const links = React.useMemo(() => {
+    switch (workflowType) {
+      case WorkflowNodeType.COMPONENT:
+      case WorkflowNodeType.STATIC_ENVIRONMENT:
+      case WorkflowNodeType.MANAGED_ENVIRONMENT:
+        return [
+          <Link key="element-link" data-testid="element-link" to={elementRef}>
+            View in tab
+          </Link>,
+        ];
+      case WorkflowNodeType.BUILD:
+      case WorkflowNodeType.RELEASE:
+        return [
+          <Link key="pipeline-runs-link" data-testid="pipeline-runs-link" to={pipelinesRef}>
+            View pipeline runs
+          </Link>,
+        ];
+      case WorkflowNodeType.TESTS:
+      case WorkflowNodeType.COMPONENT_TEST:
+      case WorkflowNodeType.APPLICATION_TEST:
+        return [
+          <Link key="element-link" data-testid="element-link" to={elementRef}>
+            View in tab
+          </Link>,
+          <Link key="pipeline-runs-link" data-testid="pipeline-runs-link" to={pipelinesRef}>
+            View pipeline runs
+          </Link>,
+        ];
+      default:
+        return null;
+    }
+  }, [elementRef, pipelinesRef, workflowType]);
+
+  return (
+    <div className="hacbs-workload-node__tooltip">
+      <div className="hacbs-workload-node__tooltip-title">{label}</div>
+      <div className="hacbs-workload-node__tooltip-description">
+        {TYPE_DESCRIPTIONS[workflowType]}
+      </div>
+      {children?.length ? (
+        <div className="hacbs-workload-node__tooltip-status-area">
+          {children
+            .filter((c) => !c.data.hidden && !c.data.isDisabled)
+            .map((child) => (
+              <React.Fragment key={child.id}>
+                <span data-testid="child-row">{child.label}</span>
+                <span>
+                  {child.data.status ? (
+                    <>
+                      <span
+                        className={css(
+                          'hacbs-workload-node__tooltip-status-icon',
+                          getRunStatusModifier(statusToRunStatus(child.data.status)),
+                        )}
+                      >
+                        <StatusIcon status={statusToRunStatus(child.data.status)} />
+                      </span>{' '}
+                      {child.data.status}
```

Comment:
Pending looks odd. But likely due to an error in a component outside this PR.
![image](https://user-images.githubusercontent.com/14068621/201208485-6150a345-9d8b-40c3-b508-da947a37f1ba.png)

Reply:
Running icon doesn't spin and also grey in color.

<img width="320" alt="image" src="https://user-images.githubusercontent.com/9964343/201662465-53922543-b09a-4c3c-ad2c-47dd807cdca3.png">